<ui version="4.0" >
 <class>MainWin</class>
 <widget class="QDialog" name="MainWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenChrono - Main window</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>2</x>
     <y>78</y>
     <width>317</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="2" column="0" >
     <widget class="QLabel" name="lbl2LastTime" >
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text" >
       <string>Last lap time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLabel" name="lblBestLapTime" >
      <property name="text" >
       <string>00:00:000</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="lbl2CurrTime" >
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text" >
       <string>Current lap time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="lblCurrLapTime" >
      <property name="font" >
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>00:00:000</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="lbl2BestTime" >
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text" >
       <string>Best lap time</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="lblLastLapTime" >
      <property name="text" >
       <string>00:00:000</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl2Lap" >
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Lap</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLap" >
        <property name="text" >
         <string>1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl2Etap" >
          <property name="font" >
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Etap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblEtap" >
          <property name="text" >
           <string>01 / 02</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="1" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl2Tmp1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>T 1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTemp1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>42</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>50°C</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl2Temp2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>T 2</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTemp2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>42</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>250°C</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblMessage" >
   <property name="geometry" >
    <rect>
     <x>2</x>
     <y>2</y>
     <width>317</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>OpenChrono</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRPMMeter" name="GraphicRPM" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>40</y>
     <width>317</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="maximum" >
    <number>16000</number>
   </property>
   <property name="value" >
    <number>11000</number>
   </property>
   <property name="alignment" >
    <set>Qt::AlignHCenter</set>
   </property>
   <property name="textVisible" >
    <bool>true</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance" >
    <bool>false</bool>
   </property>
   <property name="format" >
    <string>%v RPM</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRPMMeter</class>
   <extends>QProgressBar</extends>
   <header>qrpmmeter.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
